bip-0141.mediawiki

```diff
@@ -123,14 +123,14 @@ Sigops per block is currently limited to 20,000. We change this restriction as f
 Sigops in the current pubkey script, signature script, and P2SH check script are counted at 4 times their previous value.
 The sigop limit is likewise quadrupled to ≤ 80,000.
 
-In addition, opcodes within the witness program are counted identical to as previously within the P2SH check script.
-That is, CHECKSIG in a witness program is counted as only 1 sigop, and CHECKMULTISIG in a witness program is counted as 1 to 20 sigops according to the arguments. This rule applies to both native witness program and P2SH witness program.
+In addition, opcodes within the P2WPKH/P2WSH program are counted identical to as previously within the P2SH check script.
+That is, CHECKSIG in a P2WPKH/P2WSH program is counted as only 1 sigop, and CHECKMULTISIG in a P2WSH program is counted as 1 to 20 sigops according to the arguments. This rule applies to both native witness program and P2SH witness program.
 
```

Comment:
I suggest we use the term "witness script" rather than "P2WPKH/P2WSH program" when talking about the unhashed sequence of op codes and data pushes that then get hashed intp a P2WSH witness program. In the case of P2WPKH, the witness program is the pubkey hash and the script is implicit.